### PR TITLE
add uaa release to vbox for use with topgun tests

### PR DESCRIPTION
**Pull request by kcboyle**
Signed-off-by: Jeremy Alvis <jalvis@pivotal.io>

required for https://github.com/concourse/concourse/pull/5255

### CODE REVIEW

File: dev/vbox/setup [hunk 1]
```diff
@@ -14,9 +14,6 @@ if VBoxManage list hostonlyifs | grep vboxnet0 >/dev/null; then
     abort "'vboxnet0' interface has DHCP enabled; please disable it"
   fi
 
-  if ! VBoxManage list hostonlyifs | grep -A3 vboxnet0 | grep 'IPAddress:\s\+192.168.50.1' >/dev/null; then
```
Comment (cirocosta) [inline dev/vbox/setup]: heey, was this removal intended? wondering if that's leftovers from local changes :grin:

Comment (iplay88keys) [reply dev/vbox/setup]: Oh, ya... That was just to get us unblocked locally when testing. Feel free to revert that change.

Comment (jtarchie) [reply dev/vbox/setup]: force pushed a fix to that commit @cirocosta
